examples/local.js

```diff
@@ -19,7 +19,9 @@ th.run({
     // keep these to run a private testnet.
     name: 'lpTestNet',
     networkId: 54321,
-    controllerAddress: '0xA1fe753Fe65002C22dDc7eab29A308f73C7B6982' //pm
+    controllerAddress: '0xA1fe753Fe65002C22dDc7eab29A308f73C7B6982', //pm
+    faucet: true,
```

Comment:
if we're adding `faucet` and `txFiller` to the config files, we should check for them in `networkcreator`  instead of using `hasGeth` only